pkg/ovndbcluster/dnsdata.go

```diff
@@ -0,0 +1,74 @@
+package ovndbcluster
+
+import (
+	"context"
+	"fmt"
+
+	infranetworkv1 "github.com/openstack-k8s-operators/infra-operator/apis/network/v1beta1"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+	"github.com/openstack-k8s-operators/ovn-operator/api/v1beta1"
+	ovnv1 "github.com/openstack-k8s-operators/ovn-operator/api/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func DnsData(
+	ctx context.Context,
+	helper *helper.Helper,
+	svc *corev1.Service,
+	ip string,
+	instance *ovnv1.OVNDBCluster,
+	ovnPod corev1.Pod,
+	serviceLabels map[string]string,
+) error {
+	// ovsdbserver-(sb|nb) entry
+	headlessDnsHostname := ServiceNameSB + "." + instance.Namespace + ".svc"
```

Comment:
same can pass serviceName to the function from the caller and avoid DBType Logic here.

Reply:
Done